### DISCUSSION TITLE
How did you get that layout of the different panes in obsidian?

### DISCUSSION THREAD

**Discussion by liutiming** [BryanJenksCommunity/FAQ#5] [Q&A] [answered]
### Question

What is your question?

Hi Bryan, awesome videos so thank you so much for those! 

How did you get the nice layout of the different panes in obsidian? Referring to three on the right and the bottom left panes in this video

https://www.youtube.com/watch?v=Ewhfok91AdE&list=WL&index=1

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

[a comment by an automated account is not shown]

**Comment by tallguyjenks** [comment, marked as answer]
Each of the section tabs:

![image](https://user-images.githubusercontent.com/29872822/98970084-64e74480-24c4-11eb-8db4-8cc0b541c7c5.png)

Can be dragged around in the side panels to form your own personal layouts 🙂

**Comment by liutiming** [comment]
Fabulous and thanks! A quick comment on your current state-of-art workflow: have you tried [memex](https://getmemex.com/)? it is open sourced raindrop.io alternative and thought you might like that :p

**Comment by tallguyjenks** [comment]
@liutiming See #6 :)
